man/systemd.resource-control.xml

```diff
@@ -691,6 +691,19 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>AllowBindPorts=<replaceable>IP_PORT</replaceable></varname></term>
+
+        <listitem>
+          <para>Allow binding socket address only to certain ip ports specified by <replaceable>IP_PORT</replaceable>.
+          Same ports will be allowed for both IPv4 and IPv6 protocols. Port value must be in [0, 65535] range.
+          Special value <constant>none</constant> is reserved to block binding to any port.
+          </para>
+
+        <para>The feature is implemented with <constant>cgroup/bind{4,6}</constant> cgroup-bpf hooks.</para>
```

Comment:
Also add some words at the end about what the default value is.

Reply:
@anitazha 
By default all ports are allowed.

Reply:
@anitazha 
Won't bother CIs with doc update -- will go force push with code changes.